app/src/components/search/content.tsx

```diff
@@ -1,14 +1,27 @@
-import { Box, Menu, Text } from "@nypl/design-system-react-components";
-import { mockCollections } from "__tests__/__mocks__/data/mockCollections";
-import { useSearchParams } from "next/navigation";
-import { useRouter } from "next/navigation";
+import {
+  Box,
+  Menu,
+  Text,
+  Pagination,
+} from "@nypl/design-system-react-components";
+import { useSearchParams, usePathname, useRouter } from "next/navigation";
 import { CollectionsGrid } from "../grids/collectionsGrid";
+import React, { useState } from "react";
+import { ItemsGrid } from "../grids/itemsGrid";
 
-const SearchContent = () => {
+const SearchContent = ({ showFilter, isSearchPage, data }) => {
+  const isLoaded = true;
   const queryParams = useSearchParams();
   const query = queryParams.toString();
 
   const router = useRouter();
+  const pathname = usePathname();
```

Comment:
Since you removed the pathname logic check, I think you can remove this entirely.

Reply:
I think we'll need this for the pagination but I'll deal with this later